app/src/main/java/pl/droidsonroids/droidsmap/feature/room/business_logic/RoomFeatureUseCase.kt

```diff
@@ -1,5 +1,9 @@
 package pl.droidsonroids.droidsmap.feature.room.business_logic
 
-class RoomFeatureUseCase : RoomFeatureBoundary {
-    //no-op
+import pl.droidsonroids.droidsmap.feature.office.business_logic.OfficeFeatureBoundary
+
+open class RoomFeatureUseCase(private val officeFeatureBoundary: OfficeFeatureBoundary) : RoomFeatureBoundary {
+    override fun onRoomPerspectiveGained() {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
dałbym po prostu = TODO()
IMHO tekst `"not implemented"` nic nie wnosi, wynika to z samego TODO